scripts/run_all_benchmarks.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+# Runs all benchmarks for all pallets, for the Substrate node.
+# Should be run on a reference machine to gain accurate benchmarks
+# current reference machine: https://github.com/paritytech/substrate/pull/5848
+
+echo "[+] Running all benchmarks for Substrate"
+
+if [ $1 != "skip-build" ]
+then
+  cargo +nightly build --profile production --locked --features=runtime-benchmarks
```

Comment:
Why build using nightly?

Reply:
@coderobe any reason on your side? if not, lets get rid of it :)

Reply:
this was inherited from elsewhere from before my times - let's get rid of it then